crates/biome_js_analyze/src/lint/nursery/no_array_constructor.rs

```diff
@@ -0,0 +1,96 @@
+use biome_analyze::{
+    context::RuleContext, declare_rule, Ast, FixKind, Rule, RuleDiagnostic, RuleSource,
+};
+use biome_console::markup;
+use biome_js_syntax::{AnyJsCallArgument, AnyJsExpression, JsCallArguments, JsExpressionStatement};
+use biome_rowan::AstNode;
+
+declare_rule! {
+    /// Disallow Array constructors.
```

Comment:
Can you provide a bit more documentation about this rule? Why it exists, for example

Reply:
Yes, I quote from [eslint docs](https://eslint.org/docs/latest/rules/no-array-constructor#:~:text=Use%20of%20the,single%20numeric%20argument.).